### PR TITLE
Avoid S.S.Permissions dependency in S.S.Crypto.Xml

### PR DESCRIPTION
**Pull request by ViktorHofer**
Avoid the S.S.Permissions dependency by referencing the
latest available System.Security.AccessControl package
which contans the Evidence types in all tfms in the
System.Security.Cryptography.Xml project.

FYI @dougbu that's the accompanying change to avoid
the System.Security.Permissions dependency in main.

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -103,7 +103,7 @@
     <SystemRuntimeInteropServicesVersion>4.3.0</SystemRuntimeInteropServicesVersion>
     <SystemRuntimeInteropServicesRuntimeInformationVersion>4.3.0</SystemRuntimeInteropServicesRuntimeInformationVersion>
     <SystemRuntimeSerializationPrimitivesVersion>4.3.0</SystemRuntimeSerializationPrimitivesVersion>
-    <SystemSecurityAccessControlVersion>5.0.0</SystemSecurityAccessControlVersion>
+    <SystemSecurityAccessControlVersion>6.0.0-rc.2.21454.1</SystemSecurityAccessControlVersion>
```
Comment (ericstj) [inline eng/Versions.props:106]: This is OK for now, but we should make sure to remember to update to 6.0.0 stable when available.

Comment (ericstj) [reply eng/Versions.props:106]: One other place that might be interesting to pick up 6.0.0 prerelease is here: https://github.com/dotnet/runtime/blob/06b9703f229bc34a81e7de5743fc492d8dda9f34/src/libraries/shims/ApiCompat.proj#L8

**Review by ericstj — APPROVED** [review]

**Comment by ViktorHofer** [comment]
Failures are:
- https://github.com/dotnet/runtime/issues/58744 (reverted)
- https://github.com/dotnet/runtime/issues/11063 (xunit collection modified)
- transient NuGet Gateway timeouts

**Comment by dougbu** [comment]
I need to wait for a successful dotnet-runtime-official build and a merged dependency update PR in dotnet/aspnetcore before I can put up a reaction PR. Looks like that pipeline hasn't passed in the last few builds. Watching&hellip;

**Comment by ericstj** [comment]
My arcade changes for ARM64 broke official build and should be fixed with https://github.com/dotnet/arcade/pull/7845

**Pull request merged**